src/index.ts

```diff
@@ -199,7 +199,7 @@ export async function getMetamask(
       }
       await metamaskPage.reload()
 
-      const button = await metamaskPage.waitForSelector('.request-signature__footer__sign-button')
+      const button = await metamaskPage.waitForSelector('.button:nth-child(2)')
```

Comment:
```suggestion
      const button = await metamaskPage.waitForSelector('.signature-request-footer button:last-child')
```
race with old selector since new selector isn't backwards compatible